.azure/pipelines/jobs/default-build.yml

```diff
@@ -84,11 +82,10 @@ jobs:
       name: Hosted Ubuntu 1604
       vmImage: ubuntu-16.04
     ${{ if and(eq(parameters.poolName, ''), eq(parameters.agentOs, 'Windows')) }}:
-      ${{ if ne(parameters.codeSign, 'true') }}:
-        name: Hosted VS2017
-        vmImage: vs2017-win2016
-      ${{ if eq(parameters.codeSign, 'true') }}:
-        name: DotNetCore-Windows
+      ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+        name: dotnet-internal-temp
```

Comment:
Assuming this needs the new updated workloads, Trevor from Microbuild team is still working on making this work; Per Trevor the Microbuild plugin will be fixed "most likely in the evening once I get it written, tested, deployed, etc."

Reply:
Yes, the internal queues will need the new workload. We've been waiting on this for a while, so waiting a day more won't kill us. Thanks for the heads up.

Reply:
The Microbuild plugin rollout was successful, I'm checking to see if the new VHD works.